src/firefly/java/edu/caltech/ipac/firefly/server/persistence/QueryVOTABLE.java

```diff
@@ -44,11 +45,12 @@ public DataGroup fetchDataGroup(TableServerRequest req) throws DataAccessExcepti
             File votable = getSearchResult(getQueryString(req), getFilePrefix(req));
             DataGroup[] groups = VoTableReader.voToDataGroups(votable.getAbsolutePath(), false);
             DataGroup dg;
-            if (groups.length < 1) {
+            int tblIdx = req.getIntParam(TBL_INDEX);
```

Comment:
Bug. Should be req.getIntParam(TBL_INDEX, 0) The default value is Integer.MIN_VALUE

Reply:
Thanks.  I missed that.